squad-server/plugins/discord-admin-request.js

```diff
@@ -123,9 +134,32 @@ export default class DiscordAdminRequest extends DiscordBasePlugin {
 
     await this.sendDiscordMessage(message);
 
-    await this.server.rcon.warn(
-      info.player.steamID,
-      `An admin has been notified, please wait for us to get back to you.`
-    );
+    const admins = await this.server.getAdminsWithPermission('canseeadminchat');
+    let amountAdmins = 0;
+    if (this.options.warnInGameAdmins) {
+      for (const player of this.server.players) {
+        if (!admins.includes(player.steamID)) continue;
+        amountAdmins++;
+        await this.server.rcon.warn(player.steamID, `[${info.player.name}] - ${info.message}`);
+      }
+    }
+
+    if (amountAdmins === 0 && this.options.showInGameAdmins)
```

Comment:
I don't think this works. As it stands current `amountAdmins` will always be zero if `warnInGameAdmins` is disabled. You're better removing the first if statement and reading it on the warn, like this:

```js
    let amountAdmins = 0;
    for (const player of this.server.players) {
      if (!admins.includes(player.steamID)) continue;
      amountAdmins++;
      if (this.options.warnInGameAdmins) await this.server.rcon.warn(player.steamID, `[${info.player.name}] - ${info.message}`);
    }
```

Reply:
oh dang my bad 😞